utt/util.py

```diff
@@ -44,8 +48,42 @@ def entries_from_file(filename):
         pass
 
 
+def localize(dt):
+    """ Given a naive datetime, localize it with the machine local timezone.
+
+    Parameters
+    ----------
+    dt : datetime.datetime
+        Naive datetime.
+
+    Returns
+    -------
+    datetime : datetime.datetime
+        Timezone-aware datetime for the local timezone.
+        If failed to obtain a local timezone, UTC is assumed.
+
+    Raises
+    ------
+    pytz.InvalidTimeError
+        If the given datetime is ambiguous or does not exist for the local
+        time zone,
+    """
+    try:
+        # Avoid global cache in tzlocal
+        tz = tzlocal.reload_localzone()
```

Comment:
`get_localzone` (using the cache) should be enough given it is highly unlikely the timezone is changed for real during the lifespan of the Python process. Reloading the system file each time `localize` is called would be inefficient.